src/core/generators/client.ts

```diff
@@ -90,6 +91,50 @@ const GENERATOR_CLIENT = {
   },
 };
 
+export const addGeneratorClient = (
```

Comment:
I am not a big fan to expose a function. Orval is primarily a tool and cli. Could we maybe investigate a solution passing by the orval config? Like if you provide a function to the client property and we pass in arg the `GENERATOR_CLIENT`?

Reply:
I agree. It's a kind of hot fix, which empowers the abilities. It doesn't ruin anything, it's quite safe to use in API (not in CLI). And it can help for my usage or whoever wants to customize output. It even does have gitignore, which is a dirty hack, relying on the fact, that it won't be called by people who don't know about this little hacky feature.

The neat solution with clientGenerator could look like a path of a file with `module.exports = {client :..., msw :...}`, injected to overrides section. And that could be used as a CLI solution.

I understand all disadvantages and problems of maintaining FOSS (that's why I do contribute sometimes). 

Anyway, this little hack doesn't break anything and adds nice ability. And could be assisted with an `@experimental` tag or whatever. Like "don't rely on this, interface can be changed". But it could help people, i suppose. Probably to develop their own builder and probably, contribute in easier way.

Reply:
I can easily do the one in the orval config too if you want. I prefer to that then add hidden hack

Reply:
I opened the #142 if you want to check